NAMESPACE

```diff
@@ -36,7 +36,7 @@ export(R7_inherits)
 export(R7_object)
 export(S4_register)
 export(as_class)
-export(check_R7_inherits)
+export(check_is_R7)
```

Comment:
I guess the idea behind this naming scheme is that we typically do:

```r
is_foo <- function(x) inherits(x, "foo")
```

So `check_is_R7()` maps to `is_foo()` and `R7_inherits()` maps to `inherits()` (for now)

Reply:
Yeah, and `check_is_R7()` sounds right in my head. Could possible be `stopifnotR7()` but I don't think there's much prior art in that direction in base.